.github/workflows/release.yaml

```diff
@@ -49,3 +49,9 @@ jobs:
           args: release --rm-dist --release-notes=changes
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Invoke workflow for OLM
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: OLM bundle and PR
+          token: ${{ secrets.CR_TOKEN }}
+          inputs: '{ "bundleVersion": "master" }'
```

Comment:
shouldn't we have a `tag` here?

Reply:
both would work, if `master` is set, it will take the code from master (which is the same as the release tag). The current tag is not available anywhere and I didn't want to do something like [this](https://stackoverflow.com/a/58178121) or [this](https://github.com/k8gb-io/k8gb/blob/master/.github/workflows/cut_release.yaml#L31:L37) again. But I can add that if you think it can help with readability.